resources/lang/en_US/additional.po

```diff
@@ -105,6 +105,27 @@ msgstr "Unable to find account"
 msgid "oauth.provider-not-found"
 msgstr "Unable to find OAuth provider"
 
+msgid "oauth.invalid_request"
+msgstr "The OAuth-Provider rejected the request due to a missing or invalid parameter"
+
+msgid "oauth.unauthorized_client"
+msgstr "Not authorized as a client with the OAuth-Provider"
+
+msgid "oauth.access_denied"
+msgstr "The resource owner or authorization server denied the request"
+
+msgid "oauth.unsupported_response_type"
+msgstr "The OAuth-Provider does not support obtaining an authorization code using this method."
```

Comment:
We should either use a `.` at the end (better) or not but being inconsistent feels wrong